oauth2.md

```diff
@@ -148,10 +148,26 @@ The above example contains a request with the following parameters
 
 ```json
 {
-   "access_token": "AEjHL4Petp2EbF7GMlEQsUd5KYRVPKfOCogEzg-mHryAT",
+   "access_token": "AEjHL4Petp2EbF7GMlEQsUd5KYRVPKfOCogEzg-mHryAT...",
    "token_type": "bearer"
 }
 ```
+#### Access Token Content
+
+The access token is a JWT token that contains the following fields:
+```json
+{
+  "realmId": "random.moca",
+  "userId": "6ddc2453-2172-4ba4-b8b0-afade2cdbd64",
+  "partnerId": "421c7962-f04c-4429-b86b-2fc140a001e6",
+  "iat": 1714369562,
+  "exp": 1714455962
+}
```

Comment:
do we want to publicize this info?

Reply:
I mean it is a jwt, public by design. 😅
We can also just mention that the token contains realmId and userId without explicitly showing the payload here?

Reply:
not sure if this payload would change. I think we can say realmId would be there